manager/integration/tests/common.py

```diff
@@ -1694,8 +1694,10 @@ def wait_for_volume_faulted(client, name):
 
 
 def wait_for_volume_status(client, name, key, value,
-                           retry_count=RETRY_COUNTS):
-    wait_for_volume_creation(client, name)
+                           retry_count=RETRY_COUNTS,
+                           check_volume_creation=True):
+    if check_volume_creation is True:
+        wait_for_volume_creation(client, name)
```

Comment:
Why we need to do this? `wait_for_volume_creation` just checked if the volume exists or not, so why we have an option to ignore this check?

Reply:
Origin idea was thinking to minimize process time before catching volume faulted status, after did some test on pipeline for run test_cloning_interrupted 10 times below

- Do not have [wait_for_volume_creation](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/1816/) : 8 Pass, 2 Fail 
- Have [wait_for_volume_creation](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/1817/)            : 7 Pass, 3 Fail

Test result were almost the same no matter this part of code changed or not, I modified PR keep this part no changed, thank you

Reply:
I see. This modification is not really meaningful. We should just keep this instead of making a special case for this kind of tunning.